### ISSUE TITLE
Add a discussion of best response and player beliefs on pg. 5 

### ISSUE THREAD

**Issue by akanakia** [akanakia/SigmoidTaskAllocation#2]
Add in the common definition of what "best response" is since the target audience might not be familiar with common game theory concepts. Tie in the BR equation to player beliefs and the definition of a "Bayesian Game."

Doing this will alleviate the following reviewer concerns:
Reviewer 1:
- Page 5 beginning (end of Sect 2): this part is particularly hard to follow. I suggest the authors to explain with words the meaning of the above formula in full (specifically the part s(x) \in argmax ... is particularly obscure to me).

Reviewer 2:
Assuming that all agents know the total number of other agents sound like a strong assumption to me.

Reviewer 3:
3) The description of modeling the task allocation process as global game not clear. In Bayesian games, which are used as the main formal model in this paper, the uncertainty in other players' actions is modeled as players' types. The type of the player then determines the action that each player takes. Each player is assumed to know the joint distribution over types of all players. The modeling used by the authors does not seem to follow this canonical model. It is not clear what the notion of a player's type in the proposed model would be. In the proposed model, it looks like the type and action are combined into one parameter as the strategy function. The authors should elucidate this point further. It would also be useful to give a formal description of a global game at the beginning of section 2.

3) (contd.) In Section 2, page 5, lines 4-5, the reviewer is not convinced how the parameter 'g', (the number of other agents or players that influence the utility of the the action a_i) can be realized as stimulus function, \sum s_j(x_j). Specifically, how does an agent i observe the value of \sum s_j(x_j) for all j \neq i?

**Comment by khawrazmi** [comment]
I added an explanation for best response. Also, I have responded to comment (3) of reviewer (3) in the response file and also modified the submission file accordingly. I also responded to the last comment.
